features/networking/sctp.feature

```diff
@@ -29,8 +29,14 @@ Feature: SCTP related scenarios
     And the pod named "sctpclient" becomes ready
    
     # sctpserver pod start to wait for sctp traffic
-    When I execute on the "sctpserver" pod:
-      | bash | -c  | nc -k -l 30102 --sctp |
+    When I run the :exec background client command with:
+      | pod              | sctpserver                                |
+      | namespace        | <%= project.name %>                       |
+      | oc_opts_end      |                                           |
+      | exec_command     | sh                                        |
+      | exec_command_arg | -c                                        |
+      | exec_command_arg | /usr/bin/nc -k -l 30102 --sctp            |
```

Comment:
for various reasons I prefer not using `sh` unless necessary, and if so using `bash` explicitly at least.

```
      | pod              | sctpserver            |
      | namespace        | <%= project.name %>   |
      | oc_opts_end      |                       |
      | exec_command     | /usr/bin/nc           |
      | exec_command_arg | -k                    |
      | exec_command_arg | -l                    |
      | exec_command_arg | 30102                 |
      | exec_command_arg | --sctp                |

```